### PR TITLE
Update: cloud query search result

### PR DESCRIPTION
**Pull request by jping0929**

### CODE REVIEW

File: specs/+pcsearchresult.spec [hunk 1]
```diff
@@ -10,6 +10,16 @@ model:
 # Attributes
 attributes:
   v1:
+  - name: internalEdges
+    description: The edges of the map.
```
Comment (ericrpowers) [inline specs/+pcsearchresult.spec:14]: Same description as `publicEdges` (line 49). Maybe we can distinguish a bit more?

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "RQL-cloudquery",
    "component": "backend",
    "pr-id": "1151",
    "commit-sha": "9137bd8c1d5ea542b912187c227b0ac80df62649",
    "pipeline": "master"
  },
  {
    "project": "RQL-cloudquery",
    "component": "gaia",
    "pr-id": "642",
    "commit-sha": "26ca6cbeb018c11af9cbe244e8df7a55124a85fa",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "RQL-cloudquery",
    "component": "backend",
    "pr-id": "1151",
    "commit-sha": "3e03859670903922892e1dfd20b24938a4ffc813",
    "pipeline": "master"
  },
  {
    "project": "RQL-cloudquery",
    "component": "gaia",
    "pr-id": "642",
    "commit-sha": "6e8f343281189c846bb659c8e5bbf5efc185899f",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "RQL-cloudquery",
    "component": "backend",
    "pr-id": "1151",
    "commit-sha": "c42a3a6b507598965347a9d9d8739e9c0f7b72e4",
    "pipeline": "master"
  },
  {
    "project": "RQL-cloudquery",
    "component": "prismacloud",
    "pr-id": "28",
    "commit-sha": "d43807c3c7b4410ec885153d7aa8e7b0e79597e6",
    "pipeline": "master"
  },
  {
    "project": "RQL-cloudquery",
    "component": "gaia",
    "pr-id": "642",
    "commit-sha": "6e8f343281189c846bb659c8e5bbf5efc185899f",
    "pipeline": "master"
  }
]
```

**Review by ericrpowers — APPROVED** [review]

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "RQL-cloudquery",
    "component": "backend",
    "pr-id": "1151",
    "commit-sha": "c42a3a6b507598965347a9d9d8739e9c0f7b72e4",
    "pipeline": "master"
  },
  {
    "project": "RQL-cloudquery",
    "component": "prismacloud",
    "pr-id": "28",
    "commit-sha": "c6687b3a08dffd0194b9737bfbc061602f613b00",
    "pipeline": "master"
  },
  {
    "project": "RQL-cloudquery",
    "component": "gaia",
    "pr-id": "642",
    "commit-sha": "6e8f343281189c846bb659c8e5bbf5efc185899f",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "RQL-cloudquery",
    "component": "backend",
    "pr-id": "1151",
    "commit-sha": "2fe9c003f60ddf512388172253dbdb2c4628914e",
    "pipeline": "master"
  },
  {
    "project": "RQL-cloudquery",
    "component": "prismacloud",
    "pr-id": "28",
    "commit-sha": "c6687b3a08dffd0194b9737bfbc061602f613b00",
    "pipeline": "master"
  },
  {
    "project": "RQL-cloudquery",
    "component": "gaia",
    "pr-id": "642",
    "commit-sha": "ca4d35afcd93b001799c0b7bddb7695ef3037488",
    "pipeline": "master"
  }
]
```

**Pull request merged**